### PR TITLE
feat(mme): addition of enable5gFeatures flag

### PR DESCRIPTION
**Pull request by Lkishor123**
Signed-off-by: LKishor123 <laawanya.kishor@wavelabs.ai>

## Summary
Addition of enable5gFeatures flag in mme to centralise 5G SA enabling feature.

## Test Plan
 1. UERANSIM
 2. TeraVM

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by lionelgo — APPROVED** [review]

**Review by ulaskozat — APPROVED** [review]
LGTM

**Comment by ulaskozat** [comment]
There are merge conflicts, please resolve them.

File: lte/gateway/configs/gateway.mconfig [hunk 1]
```diff
@@ -42,6 +42,7 @@
       "enable5gFeatures": false,
       "defaultSliceServiceType": "1",
       "defaultSliceDifferentiator" : "0xffffff"
+
```
Comment (ulaskozat) [inline lte/gateway/configs/gateway.mconfig]: remove extra space

Comment (Lkishor123) [reply lte/gateway/configs/gateway.mconfig]: Comment addressed.

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**